test/test_models.py

```diff
@@ -514,6 +515,35 @@ def test_generalizedrcnn_transform_repr():
     assert t.__repr__() == expected_string
 
 
+test_vit_conv_stem_configs = [
```

Comment:
Nit: No need for defining this on module level. Could move in `vitc_b_16` as it's used only once.

Reply:
Will clean this up (with several other changes) in a seperate diff!